### ISSUE TITLE
Snippets with utf8 characters

### ISSUE THREAD

**Issue by Casteran** [coq-community/hydra-battles#113]
On branch utf8 I added a snippet `utf8try`at the end of `gaia/T1Bridge.v` with the utf8 character for omega.  The file `movies/snippets/T1Bridge.tex` looks to be OK,
but in `hydras.pdf` all the occurrences of the greek letter are missing (page 206).

Is there a standard way to use utf8 characters all along the toolchain ?

**Comment by Zimmi48** [comment]
Cc @cpitclaudel

**Comment by cpitclaudel** [comment]
Probably a font issue?  The log will say something like this:

```
Missing character: There is no λ (U+03BB) in font [lmmono10-regular]:!
{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}
Missing character: There is no α (U+03B1) in font [lmmono10-regular]:!
] (./test.aux))
```

Can you check?

**Comment by Casteran** [comment]
@cpitclaudel  
Indeed, the issue happens only inside the snippets. If I type an utf8 omega in the
main tex, it is correctly displayed.

**Comment by cpitclaudel** [comment]
If you're OK with changing the font, it could be as simple as `\setmonofont{Fira Code}[Scale=MatchLowercase]` after the line that loads `fontspec` (you can install that font from https://github.com/tonsky/FiraCode).

**Comment by cpitclaudel** [comment]
IIRC `\setmonofont{Liberation Mono}` would work as well.

**Comment by Casteran** [comment]
@cpitclaudel  Thanks a lot ! 
I installed FiraCode on my machine, at least "Check ω." works!
@Zimmi48 @palmskog  Do we have to install this font somewhere for the CI/CD?

**Comment by Casteran** [comment]
`\setmonofont{DejaVu Sans Mono}[Scale=MatchLowercase]`  looks good too.

**Comment by Zimmi48** [comment]
Sorry for not responding earlier. I have done a precise listing of the LaTeX packages that are needed to compile the book in:

https://github.com/coq-community/hydra-battles/blob/8b50732058d291b03cada050d1e41a6a6061712a/.nix/coq-overlays/hydra-battles-single/default.nix#L15-L42

The advantage is that this represents less to download in CI, but the drawback of course is that this list needs to be kept up-to-date if we add dependencies on new LaTeX packages...

**Comment by Casteran** [comment]
Thanks @Zimmi48 !
Do we have to include this list in the installation directives for the users (as well as the specific fonts needed for compiling the utf8 snippets) ?

**Comment by Zimmi48** [comment]
> Do we have to include this list in the installation directives for the users (as well as the specific fonts needed for compiling the utf8 snippets) ?

Sure, that could make sense, although lots of users will just have a "full TeXLive" distribution, which is always sufficient.
